eng/azure-pipelines.yml

```diff
@@ -188,6 +235,14 @@ stages:
           inputs:
             artifact: Artifacts_Browser_Threading
             path: 'artifacts/bin/browser-threading'
+        - task: DownloadPipelineArtifact@2
```

Comment:
For a future PR: This `Publish Packages` job should run for PRs too, and generate the packages, but skip the *publishing* part itself. This would ensure that the PR doesn't break packaging.